zeebe/engine/src/test/java/io/camunda/zeebe/engine/util/client/MappingClient.java

```diff
@@ -61,6 +61,11 @@ public MappingCreateClient withClaimValue(final String claimValue) {
       return this;
     }
 
+    public MappingCreateClient withId(final String id) {
+      mappingRecord.setId(id);
+      return this;
+    }
+
```

Comment:
🔧 As id is the identifier I'd prefer having a method `withClaimName(String)` and putting id in the `newMapping(String)` method. So it'd become:

```java
engine.mapping()
  .newMapping(id)
  .withClaimName(claimName)
  .withClaimValue(claimValue)
  .withName(name)
  .create()
```

WDYT?

Reply:
The reason that I didn't go for this option was that already there are a lot usage (34 times) for newMapping(name) method and I didn't want to touch all these files in this PR.

Reply:
I think it's a valid refactoring to do. If you keep it isolated in a single commit it doesn't make the PR harder to review to me 🤷

Reply:
I'll do it in a separate PR then.